src/background.js

```diff
@@ -25,7 +26,8 @@ async function createWindow() {
 
 	win = new BrowserWindow({
 		title: 'LISTEN.MOE - Desktop App',
-		icon: 'public/logo.png',
+		// eslint-disable-next-line no-undef
+		icon: join(__static, 'logo.png'),
```

Comment:
This ensures the application logo is loaded for other windows too, it's a weird one.